src/components/Footer.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <footer>
-    <nav aria-label="Footer">
+    <nav>
```

Comment:
I think the `nav`s should be changed to `div`s if Ernesto wants us to get rid of the navigation landmark. You'll also have to check that no styles depend on it being a nav tag and if so change them

Reply:
You can ask him about it, seems strange to me too but he's done it this way for other applications (Angular ones)